template/test/unit/.eslintrc

```diff
@@ -1,9 +1,5 @@
 {
   "env": {
-    "mocha": true
```

Comment:
since we are leaving the karma/mocha setup in place, the eslint config should probably also leave this in.

It should be conditional:
```javascript
{
  "env": {{{#if_eq runner "karma"}}
    "mocha": true{{/if_eq}}{{#if_eq runner "jest"}}
    "jest"{{/if_eq}}
  }{{#if_eq runner "karma"}},
  "globals": {
    "expect": true,
    "sinon": true,
    "jest": true
  }{{/if_eq}}
}
```